packages/test/mocha-test-setup/mocharc-common.js

```diff
@@ -78,8 +78,17 @@ function getFluidTestMochaConfig(packageDir, additionalRequiredModules, testRepo
 
 	if (process.env.FLUID_TEST_MULTIREPORT === "1") {
 		config["reporter"] = `mocha-multi-reporters`;
-		config["reporter-options"] = [`configFile=${path.join(__dirname, "test-config.json")}`];
-		console.log(`configFile=${path.join(__dirname, "test-config.json")}`);
+		// See https://www.npmjs.com/package/mocha-multi-reporters#cmroutput-option
+		const outputFilePrefix = testReportPrefix !== undefined ? `${testReportPrefix}-` : "";
+		console.log(
+			`Writing test results relative to package to nyc/${outputFilePrefix}junit-report.xml and nyc/${outputFilePrefix}junit-report.json`,
+		);
```

Comment:
Can we create a follow-up item to make this actually tracks what `test-config.json` says (or points out that the actual filenames are what that file says they'll be)?